db/migrations/versions/91e0d528fc9f_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
+
+Revision ID: 91e0d528fc9f
+Revises: 190bf5378715
+Create Date: 2022-11-24 15:52:01.294846
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '91e0d528fc9f'
+down_revision = '5ce0b8e0e1b6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('just_scores',
+    sa.Column('score_id', postgresql.UUID(as_uuid=True), nullable=False),
+    sa.Column('score', sa.Integer(), nullable=False),
+    sa.Column('justification', sa.Text(), nullable=False),
+    sa.Column('application_id', postgresql.UUID(), nullable=True),
+    sa.Column('timestamp', sa.DateTime(), nullable=False),
```

Comment:
Can we call this `date_created`

Reply:
Think it is safe to default this as per https://github.com/communitiesuk/funding-service-design-application-store/blob/559d62e7f08f1dc5b143283f2110fc46014803d3/db/models/applications.py#L41 and then there is no need for the client to provide it.

Reply:
ahh yes great shout! have added this now 👍🏾